### ISSUE TITLE
How to train the discriminators?

### ISSUE THREAD

**Issue by prokaryote-hgy** [gfxdisp/mdf#7]
Hi, I'm interested in your work "Training a Task-Specific Image Reconstruction Loss". I'm wondering how to train the discriminators used for calculate the MDF loss (the phase 1 mentioned in the paper) ? It seems that the training code is not provided in this repo.

**Comment by aamir-mustafa** [comment]
Hi,

Thanks a lot for your interest in our paper. Could you please refer to the issue:
https://github.com/gfxdisp/mdf/issues/1#issue-866931874

You will find a detailed conversation where I have tried to answer similar questions there.

Hope this helps.

Thanks
